datasource/myharvard.go

```diff
@@ -247,20 +261,24 @@ func mhTo24hr(s string) string {
 
 func mhMakeMeetingPattern(mon, tues, wed, thurs, fri, sat, sun,
 	startTime, endTime, startDate, endDate any) *MeetingPattern {
-	if mon.(string) == "Y" || tues.(string) == "Y" || wed.(string) == "Y" ||
-		thurs.(string) == "Y" || fri.(string) == "Y" || sat.(string) == "Y" || sun.(string) == "Y" {
+	isYes := func(value any) bool {
+		value, ok := value.(string)
+		return ok && value == "Y"
```

Comment:
The redeclaration of `value` hides the asserted `string` type, so the final line compares a value of type `any` to the string `"Y"`. It's unimportant in this tiny function, but for the sake of type inference maybe use two variables:
```suggestion
	isYes := func(valueRaw any) bool {
		value, ok := valueRaw.(string)
		return ok && value == "Y"
```

Reply:
Thanks! Yeah good point